packages/slice-machine/src/modules/selectedSlice/reducer.ts

```diff
@@ -104,6 +108,18 @@ export const selectedSliceReducer: Reducer<
         return prevState;
       }
     }
+    case getType(generateSliceScreenshotCreator.success):
+    case getType(generateSliceCustomScreenshotCreator.success): {
+      if (!prevState) return prevState;
```

Comment:
so if prevState is null it stays that way?

Reply:
it's a bit outside the scope of this PR

Reply:
true, take a note of it and put it on the tech-debt board ad maybe we'l fix it later :)